onchaind/test/run-grind_feerate-bug.c

```diff
@@ -23,9 +23,6 @@ bool derive_keyset(const struct pubkey *per_commitment_point UNNEEDED,
 		   const struct basepoints *other UNNEEDED,
 		   struct keyset *keyset UNNEEDED)
 { fprintf(stderr, "derive_keyset called!\n"); abort(); }
```

Comment:
Commit itself looks fine but why "pytest" in the commit message...?

Reply:
Oops, I "--fixup" the wrong commit :(  Will all sort out once I squash it...